cli/maintainer/src/maintenance.rs

```diff
@@ -1239,15 +1246,25 @@ impl SolidoState {
                 .map(|(_addr, stake_account)| stake_account.balance)
                 .sum();
 
+            // Stake here can only be inactive or deactivating.
+            let unstake_balance: StakeBalance = unstake_accounts
+                .iter()
+                .map(|(_addr, stake_account)| stake_account.balance)
+                .sum();
```

Comment:
I believe this works, but simpler would be to add the `unstake_balance` to the `stake_balance` and keep the metrics as-is. It’s already separated out into activating/active/deactivating/inactive, which account we found it in should not matter.

The diff could be as simple as

```diff
            let stake_balance: StakeBalance = stake_accounts
                .iter()
+               .chain(&unstake_accounts)
                .map(|(_addr, stake_account)| stake_account.balance)
                .sum();
```